nuxeo-core/nuxeo-core/src/main/java/org/nuxeo/ecm/core/migrator/AbstractRepositoryMigrator.java

```diff
@@ -37,13 +37,25 @@
 
     private static final Log log = LogFactory.getLog(AbstractRepositoryMigrator.class);
 
+    /**
+     * @deprecated since 11.1. Not used any more as a filed of this class, you should pass it as a parameter instead
```

Comment:
In fact, implementation is not always responsible of passing parameters as in some cases they are called by the `MigrationService`.

Here it's more:
```
    /*
     * @deprecated since 11.1, please use parameters when using or implementing:
     * <ul>
     *   <li>{@link #migrateRepository(String, MigrationContext, String)}</li>
     *   <li>{@link #migrateSession(String, MigrationContext, CoreSession)}</li>
     *   <li>{@link #checkShutdownRequested(MigrationContext)}</li>
     * ...
     * </ul>
     */
```
I didn't list every methods involved.